anyway/infographics_utils.py

```diff
@@ -5,6 +5,8 @@
 import os
 from functools import lru_cache
 from typing import Optional, Dict, List, Union, Any, Type, Callable
+
+# noinspection PyUnresolvedReferences
 from dataclasses import dataclass
```

Comment:
Now that i think about it - I added in the past code with dataclasses and the code is in PROD so dataclasses are definitely supported.

But i don't see any use of dataclasses in this code - possibly an unused import?

Reply:
Without the `noinspection`, it is complaining on

![image](https://user-images.githubusercontent.com/28224696/113324038-4f61a280-931f-11eb-871d-d4313af666ee.png)

It seems that it is looking for `dataclass`, in anyway.dataclasses rather than in the global `dataclasses` module. Is it possible it is because we use a module with same name as a global module?

Reply:
Yes, to be in the safe side it might be better if we change the name of the folder, i will take it on myself.